commerce_coordinator/apps/commercetools/utils.py

```diff
@@ -262,6 +262,26 @@ def find_latest_refund(payment: Payment):
     return ''
 
 
+def get_refund_transaction_id_from_mobile_order(order: Order) -> str:
+    """
+    Utility to find the refund transaction ID in a mobile order.
+
+    Args:
+        order (Order): The Commercetools order object.
+
+    Returns:
+        str: The ID of the refund transaction, or an empty string if no refund transaction exists.
+    """
+    if order.payment_info and order.payment_info.payments:
+        for paymentReference in order.payment_info.payments:
+            payment = paymentReference.obj
+            for transaction in payment.transactions:
```

Comment:
we should only check in latest payment, and orders will always have payment_info so empty check is unnecessary
```suggestion
    for transaction in order.payment_info.payments[-1].obj:
```